### DISCUSSION TITLE
Contacting users privately

### DISCUSSION THREAD

**Discussion by isitreallyalive** [community/community#21745] [New to GitHub]
<p>Out of interest, are there any ways for someone to contact another GitHub user privately? I believe it would be pretty handy to have, but I can’t find any methods of doing so through the GitHub website.</p>
<p>If there is none built into GitHub, is there any other ways I could find a way to contact a GitHub user?</p>

**Comment by Gisgar3** [comment, marked as answer]
<p>Other than viewing their email through their  <strong>GitHub </strong> profile (if it’s a public email) or visiting their website to view their contact information, I don’t think they’re any other options through  <strong>GitHub’s</strong> services. I would recommend searching their username on  <strong>Google</strong> and looking throughout the results to see if you can find a match. Hope it all goes well!</p>
